File: MPI/mpicheck/tests/functions/2.0/MPI_Type_create_f90_integer.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_type_create_f90_integer(var_0, var_1, var_2)
       call pmpi_type_create_f90_integer(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Type_create_f90_integer.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Datatype *var_1;
    int ret;
    /* calls */
    ret = MPI_Type_create_f90_integer(var_0, var_1);
    ret = PMPI_Type_create_f90_integer(var_0, var_1);
    return 0;
}
